src/Entity/LibraryTrait.php

```diff
@@ -432,4 +432,17 @@ public function belongsToMunicipalConsortium() : bool
             'regional',
         ]);
     }
+
+    public function setDefaultLangcode(string $langcode) : void
```

Comment:
Can we add a function comment? Seems like it might be useful to have one.

Reply:
Would something like " Set default language code and also set any contained addresses default language code." work?

Reply:
Yeah something like that seems like it would be OK, maybe "Set default language code and also set it for all contained addresses."

Reply:
Okay, I added the comment above to the function.